### PR TITLE
Fixes#1247 Add the test for SingUpDetail screen

### LINKED ISSUE #1247: Views: Create tests for signup_details.dart
The Talawa code base needs to be 100% reliable. This means we need to have 100% test code coverage.

Tests need to be written for file `lib/views/pre_auth_screens/signup_details.dart`

- When complete, all methods, classes and/or functions in the file will need to be tested. 
- These tests must be placed in a single file with the name `test/views/pre_auth_screens_test/signup_details_test.dart`. You may need to create the appropriate directory structure to do this.

### IMPORTANT:
Please refer to the parent issue on how to implement these tests correctly:

- https://github.com/PalisadoesFoundation/talawa/issues/1217

### PR Acceptance Criteria

- When complete this file must show **100%** coverage when merged into the code base. This will be clearly visible when you submit your PR.
- [The current code coverage for the file can be found here](https://codecov.io/gh/PalisadoesFoundation/talawa/tree/develop/lib/views/pre_auth_screens/). If the file isn't found in this directory, or there is a 404 error, then tests have not been created.
- The PR will show a report for the code coverage for the file you have added. You can use that as a guide.
- You can verify your own code coverage by creating an account at [Codecov.io](https://codecov.io)

### PR DESCRIPTION
**Pull request by Ayush0Chaudhary**
**What kind of change does this PR introduce?**

**Issue Number:**

Fixes #1247 

**Did you add tests for your changes?**
Yes, I added the unittest only

**Summary**
I added the test for sign up detail screen
**Does this PR introduce a breaking change?**
no

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa/blob/master/CONTRIBUTING.md)?**
yes

### CODE REVIEW

**Comment by Ayush0Chaudhary** [comment]
@noman2002 @palisadoes in Signup page.@SiddheshKukade @TheHazeEffect 
I got 100% coverage in Signup page.
Screenshot from 2023-01-27 05-30-15

**Please review the PR mentors.**

**Comment by palisadoes** [comment]
@Ayush0Chaudhary Please fix the failing tests. You have probably not updated your local branch with the latest version of the code base.

**Comment by Ayush0Chaudhary** [comment]
I merged the latest commit from develop to my branch @palisadoes @noman2002

**Comment by Ayush0Chaudhary** [comment]
@palisadoes @noman2002 you can check it now!!
It should pass all the test now.
@SiddheshKukade @TheHazeEffect

File: test/widget_tests/after_auth_screens/app_settings/app_setting_page_test.dart [hunk 1]
```diff
@@ -90,14 +85,14 @@ Future<void> main() async {
...
-  await Hive.openBox<OrgInfo>('currentOrg');
-  await Hive.openBox('url');
+  // final Directory dir = Directory('temporaryPath');
+  // Hive
+  //   ..init(dir.path)
+  //   ..registerAdapter(UserAdapter())
+  //   ..registerAdapter(OrgInfoAdapter());
+  // await Hive.openBox<User>('currentUser');
+  // await Hive.openBox<OrgInfo>('currentOrg');
+  // await Hive.openBox('url');
```
Comment (noman2002) [inline test/widget_tests/after_auth_screens/app_settings/app_setting_page_test.dart:95]: Why we need to make these changes??. I don't think we need these changes in this PR.

Comment (Ayush0Chaudhary) [reply test/widget_tests/after_auth_screens/app_settings/app_setting_page_test.dart:95]: It was failing the test otherwise at my local machine

Comment (noman2002) [reply test/widget_tests/after_auth_screens/app_settings/app_setting_page_test.dart:95]: But  it is not failing in any other PR??

**Comment by Ayush0Chaudhary** [comment]
@noman2002 I added them back

**Comment by Ayush0Chaudhary** [comment]
@noman2002 sir, what about this PR??
can we merge it??

**Comment by noman2002** [comment]
> @noman2002 sir, what about this PR?? can we merge it??

Yes we are good to go.

**Review by noman2002 — APPROVED** [review]

**Review by palisadoes — APPROVED** [review]

**Pull request merged**